src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {checkInput, transformTextToStringArray, eliminatePunctuation} from "./detectWrongSyntax";
+
+
+it("Works for simple text", () => {
+    const wrongWords = checkInput("fu bar bloedsinn"); //fu and bar are indeed included in allowedVocab
+    expect(wrongWords).toEqual(["bloedsinn"]);
+  });
+
+it("Works for an array containing only the empty String", () =>{
+    const wrongWords = checkInput("");
+    expect(wrongWords).toEqual([""]);
+});
+
+it("Works for an array containing duplicates", () =>{ 
+    const wrongWords = checkInput("bloedsinn bloedsinn");
+    expect(wrongWords).toEqual(["bloedsinn", "bloedsinn"]);
```

Comment:
Why do we want the output to contain duplicates? Wouldn't it suffice to output `bloedsinn` once?